board/safety/safety_honda.h

```diff
@@ -212,12 +219,12 @@ static int honda_rx_hook(CANPacket_t *to_push) {
 
     bool stock_ecu_detected = false;
     int bus_rdr_car = (honda_hw == HONDA_BOSCH) ? 0 : 2;  // radar bus, car side
```

Comment:
Is this correct? "Radar" side here is 2, but it's checking if 0xE4 (STEERING_CONTROL) is on the destination bus (which comes from bus 2), so we want it to check bus 0 if I'm not mistaken.